src/main/java/frc/robot/subsystems/elevator/ElevatorIOTalonFX.java

```diff
@@ -24,18 +24,18 @@ public class ElevatorIOTalonFX implements ElevatorIO {
 
   public ElevatorIOTalonFX(int lead, int follow) {
```

Comment:
These should probably be labeled as leadCANID or leadID so it's clear that we're passing an id, not an object